### ISSUE TITLE
Vk.xml version 1.0.64 doesn't parse

### ISSUE THREAD

**Issue by tomaka** [Osspial/vk-rs#11]
The generator panics when doing so.

```
thread 'main' panicked at 'Unexpected tag', /home/pierre/.cargo/registry/src/github.com-1ecc6299db9ec823/vk_generator-0.2.1/src/registry/crawler.rs:282:46
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::print
             at libstd/sys_common/backtrace.rs:68
             at libstd/sys_common/backtrace.rs:57
   2: std::panicking::default_hook::{{closure}}
             at libstd/panicking.rs:380
   3: std::panicking::default_hook
             at libstd/panicking.rs:396
   4: std::panicking::rust_panic_with_hook
             at libstd/panicking.rs:576
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:537
   6: vk_generator::registry::crawler::crawl
             at /home/pierre/.cargo/registry/src/github.com-1ecc6299db9ec823/vk_generator-0.2.1/src/registry/crawler.rs:282
   7: vk_generator::registry::VkRegistry::new
             at /home/pierre/.cargo/registry/src/github.com-1ecc6299db9ec823/vk_generator-0.2.1/src/registry/mod.rs:111
   8: build_script_build::main
             at vulkano/build.rs:30
   9: std::rt::lang_start::{{closure}}
             at /checkout/src/libstd/rt.rs:74
  10: std::panicking::try::do_call
             at libstd/rt.rs:59
             at libstd/panicking.rs:479
  11: __rust_maybe_catch_panic
             at libpanic_unwind/lib.rs:102
  12: std::rt::lang_start_internal
             at libstd/panicking.rs:458
             at libstd/panic.rs:358
             at libstd/rt.rs:58
  13: std::rt::lang_start
             at /checkout/src/libstd/rt.rs:74
  14: main
  15: __libc_start_main
  16: _start
```

**Comment by kvark** [comment]
@Osspial ping!

**Comment by Osspial** [comment]
Resolved by 24d2cc72b249f4a166f0b86d121cbd3d31e790a6.
